src/index.ts

```diff
@@ -555,7 +558,10 @@ dapp.setup({
   binaryDeserializeObject: null,
   verifyMultiSigs: function (rawPayload: object, sigs: ShardusTypes.Sign[], allowedPubkeys: { [pubkey: string]: ShardusTypes.DevSecurityLevel }, minSigRequired: number, requiredSecurityLevel: ShardusTypes.DevSecurityLevel): boolean {
     return false
-  }
+  },
+  beforeStateAccountFilter(account: ShardusTypes.WrappedData) {
+    return false
```

Comment:
Just to confirm. This means we are not sending any beforeStates to the archivers, right?

Reply:
Yes ... this will not send any beforeStates to the archiver.